umm-spec-lib/src/cmr/umm_spec/migration/version/service.clj

```diff
@@ -66,3 +198,23 @@
       (update :Coverage migrate-coverage-type-down)
       (dissoc :RelatedURLs)
       util/remove-empty-maps))
+
```

Comment:
Really nice job with all the migration code; I think this is the best example for migrations that I've seen in the CMR to-date.

Reply:
Thanks!